bin/autoformat

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e
+
+rubocop -a -c rubocop.yml
+npm run format-fix
```

Comment:
🐘 since we use `eslint-plugin-prettier` `eslint --fix` will be apply prettier styling too, so we don't really need to run this.